### ISSUE TITLE
Add configuration documentation for overrides module

### ISSUE THREAD

**Issue by annanay25** [grafana/tempo#971]
The configuration page (https://grafana.com/docs/tempo/latest/configuration/) needs a section on the overrides module and the various per-tenant override settings.

**Comment by kehao95** [comment]
Hi I encountered this issue yesterday. The https://grafana.com/docs/tempo/latest/configuration/ingestion-limit/ helped me out. But I agree with you that we should have `override` section in the configuration main page.

One thing I'm not sure: The original ingestion-limit page seems too gossipy to be fit in the configuration page, but also no much left if I extract the core info to the configuration documentation index page. Anyway, my attempt https://github.com/grafana/tempo/pull/985  was to simplify the article to fit into main config then I removed the old ingestion-limit page. This will keep it organized & tidy. But may risk a bit more difficulty for users to find it. Would love to hear anyone's suggestions to improve.
